example/styles.css

```diff
@@ -72,11 +72,11 @@ z-index: 10; }
     max-width: 100%;
   }
 
-  .leaflet-control-layers-scrollbar {
+  .leaflet-control-layers-menu .leaflet-control-layers-scrollbar {
     overflow-y: hidden;
   }
 
-  .leaflet-control-layers-overlays {
+  .leaflet-control-layers-menu .leaflet-control-layers-overlays {
```

Comment:
We will need this CSS as well in plots2, right?
Also we need to make sure that we use different CSS for different custom bars. Thanks!

Reply:
> We will need this CSS as well in plots2, right?

Thanks for mentioning this. I had assumed it would be bundled into dist. I think it is safe to move the styles to `LeafletEnvironmentalLayers.css` in /dist. What do you think?

Reply:
> Also we need to make sure that we use different CSS for different custom bars.

Yes, you are right. That is why I had added `.leaflet-control-layers-menu` which is a custom class so that it does not override leaflet's default `.leaflet-control-layers-overlays`.